pyzipcode/zipcode.csv

```diff
@@ -37952,33 +37952,33 @@ zip,city,state,latitude,longitude,timezone,dst
 86016,Gray Mountain,AZ,35.675689,-111.49474,-7,0
 86017,Munds Park,AZ,34.9372,-111.62931,-7,0
 86018,Parks,AZ,35.197566,-111.93253,-7,0
-86020,Cameron,AZ,36.182818,-111.5472,-7,0
+86020,Cameron,AZ,36.182818,-111.5472,-7,1
 86021,Colorado City,AZ,36.985868,-112.9791,-7,0
 86022,Fredonia,AZ,36.844685,-112.47107,-7,0
 86023,Grand Canyon,AZ,35.862292,-112.06498,-7,0
 86024,Happy Jack,AZ,34.690846,-111.16676,-7,0
 86025,Holbrook,AZ,34.936261,-110.14698,-7,0
 86028,Petrified Forest Natl Pk,AZ,35.237487,-109.52295,-7,0
-86029,Sun Valley,AZ,35.285746,-110.288704,-7,0
+86029,Sun Valley,AZ,35.285746,-110.288704,-7,1
 86030,Hotevilla,AZ,36.115339,-110.61562,-7,0
-86031,Indian Wells,AZ,35.48041,-110.09228,-7,0
+86031,Indian Wells,AZ,35.48041,-110.09228,-7,1
 86032,Joseph City,AZ,35.101925,-110.42509,-7,0
-86033,Kayenta,AZ,36.608485,-110.18425,-7,0
-86034,Keams Canyon,AZ,35.765669,-110.18617,-7,0
-86035,Leupp,AZ,35.335107,-110.99972,-7,0
+86033,Kayenta,AZ,36.608485,-110.18425,-7,1
+86034,Keams Canyon,AZ,35.765669,-110.18617,-7,1
+86035,Leupp,AZ,35.335107,-110.99972,-7,1
 86036,Marble Canyon,AZ,36.618595,-111.85444,-7,0
 86038,Mormon Lake,AZ,34.908287,-111.463026,-7,0
 86039,Kykotsmovi Village,AZ,36.00059,-110.5201,-7,0
-86040,Page,AZ,36.726377,-111.4284,-7,0
+86040,Page,AZ,36.726377,-111.4284,-7,1
```

Comment:
Page, Arizona doesn't observe DST according to https://www.timetemperature.com/tzaz/page.shtml

The coordinate is quite a ways south of the town.